pkg/platform/kube/functionres/lazy.go

```diff
@@ -2338,6 +2334,39 @@ func (lc *lazyClient) isPodAutoScaledUp(ctx context.Context, pod v1.Pod) (bool,
 	return false, nil
 }
 
+func (lc *lazyClient) populateDefaultContainerResources(ctx context.Context, container *v1.Container) {
+
+	defaultFunctionPodResources := lc.platformConfigurationProvider.GetPlatformConfiguration().Kube.DefaultFunctionPodResources
+
+	lc.logger.DebugWithCtx(ctx, "Populating container resources with default values",
+		"defaultFunctionPodResources", defaultFunctionPodResources)
+
+	if container.Resources.Requests == nil {
```

Comment:
if `defaultFunctionPodResources` is also empty, use hard coded request - `container.Resources.Requests["cpu"] = apiresource.MustParse("25m")`

Reply:
Hard code it only for requests' cpu or for the other values as well?